### ISSUE TITLE
Audio and Video are not working when connecting from MAC to Windows in Docker jitsi stable

### ISSUE THREAD

**Issue by usharaniyerra** [jitsi/docker-jitsi-meet#1484]
Hi Team,

I am using docker jitsi with stable version 8252. When we initiate a call from windows to windows audio and video are working properly.

But when we try to connect from Mac system to windows audio and video both are not working. 

Please suggest, do i have to add any extra changes in the configuration or in sip-communicator.properties file?

I have read in ubuntu setup documentation to add LOCAL_ADDRESS and PUBLIC_ADDRESS extra in sip-communicator.properties file. Have added the same in our jvb container. But still the issue is same.

Could someone please help on this issue?

![image](https://user-images.githubusercontent.com/37344123/221600915-bcfa4932-8ef8-4564-8465-80fb6c86cbe1.png)

**Comment by damencho** [comment]
> PUBLIC_ADDRESS extra in sip-communicator.properties file.

What about the port forwarding of udp port 10000 from the public address to the jvb, is that working?

**Comment by usharaniyerra** [comment]
I think it's working. Can you guide me how can I check that?

**Comment by damencho** [comment]
https://community.jitsi.org/t/tip-how-to-check-udp-10000-connectivity/73031

**Comment by usharaniyerra** [comment]
No...ngrep is running on main host. netcat is running in jvb container.

Do i need to check vice-versa?

**Comment by damencho** [comment]
ngrep should be on the jvb container you need to stop jvb in order to run it. The NC should run somewhere in the internet, not local network, should be a remote machine like a client joining from a remote internet location.

**Comment by usharaniyerra** [comment]
Yes.. This way its not working. I have checked as below.

JVB container:
![image](https://user-images.githubusercontent.com/37344123/222179486-199f81c7-d129-4587-8776-0cbdbf381a45.png)

NC from outside:
![image](https://user-images.githubusercontent.com/37344123/222179650-d5b29c85-9b42-4412-920c-7ffb891a1c08.png)


Please suggest what i have to do now? How can i resolve this issue?

**Comment by damencho** [comment]
On the device hosting the public ip you need to setup port forwarding for udp 10000 to the machine hosting jvb ...

**Comment by usharaniyerra** [comment]
Hi @damencho ,

I have done the port forwarding with the help of some documentation for Linux.

![image](https://user-images.githubusercontent.com/37344123/222220919-eec86b98-2fcd-489d-98b1-38412fc663c5.png)


But still its not working. Is there any workaround to do from jitsi configuration side like adding any extra_hosts in docker-compose yml file or any other?

I have referred many docs and added few solutions. But nothing worked.

Could you please help me with this.

**Comment by damencho** [comment]
Is that the machine that has the public address? If you do ifconfig you see the public address?

Nope, this is a network issue you need to solve and there is nothing jitsi can do to fix it for you.

**Comment by usharaniyerra** [comment]
Is that the machine that has the public address? - **Yes**

If you do ifconfig you see the public address? - **No**, i dont see public IP when i run ifconfig. But from cloud console i can see the public IP for this machine.

**Comment by damencho** [comment]
This means that its not the machine managing the IP and there is some NAT in front of it which you need to instruct to forward the port. If its cloud provider its done in their web console. In aws is called security groups or something.

**Comment by usharaniyerra** [comment]
Yes.. correct. I have checked in the security groups. The port 10000 has configured wrongly. Its has opened to be accessible within the subnet. I have changed it to Any-Any for source and destination for this port.

Now audio and video are working fine from mac to windows. Issue is resolved. :)

Thanks a lot for your support @damencho
